src/components/block-button.js

```diff
@@ -14,11 +14,11 @@ AFRAME.registerComponent("block-button", {
   },
 
   play() {
-    this.el.addEventListener("click", this.onClick);
+    this.el.addEventListener("grab-start", this.onClick);
```

Comment:
I have a TODO item to revisit the user bindings for this so that click worked like before and to make sure block-button, remove-networked-icon-button, icon-button are DRY and consistent